service/src/main/java/uk/gov/hmcts/reform/fpl/service/DraftCMOService.java

```diff
@@ -42,18 +48,93 @@ public DraftCMOService(DateFormatterService dateFormatterService, ObjectMapper m
         this.directionHelperService = directionHelperService;
     }
 
-    public DynamicList getHearingDateDynamicList(CaseDetails caseDetails) {
-        Map<String, Object> data = caseDetails.getData();
-        CaseData caseData = mapper.convertValue(data, CaseData.class);
+    public Map<String, Object> extractIndividualCaseManagementOrderObjects(
+        CaseManagementOrder caseManagementOrder,
+        List<Element<HearingBooking>> hearingDetails) {
 
-        List<Element<HearingBooking>> hearingDetails = caseData.getHearingDetails();
+        if (isNull(caseManagementOrder)) {
```

Comment:
I think doing `== null` is more readable. There is no good reason to use `isNull` method.

Not saying change it now but more a clue for the future.

Reply:
I prefer `== null` as well. I had seen `isNull` and similar around the rest of the code base so stuck with the convention.